src/services/routeMatchScore.ts

```diff
@@ -47,14 +47,14 @@ export function countExpectedPathParams(route: Route, actualPath: string): numbe
     .filter((key) => key.startsWith('?'))
     .map((key) => key)
 
-  const missing = optionalParams.filter((expected) => getParamValueFromUrl(actualPath, route.path.value, expected) === undefined)
+  const missing = optionalParams.filter((expected) => getParamValueFromUrl(actualPath, route.path.toString(), expected) === undefined)
 
   return optionalParams.length - missing.length
 }
 
 export function countExpectedQueryParams(route: Route, actual: QuerySource): number {
   const actualQuery = new URLSearchParams(actual)
-  const expectedQuery = new URLSearchParams(route.query.value)
+  const expectedQuery = new URLSearchParams(route.query.toString())
```

Comment:
Why did we go back to having a `toString` method? Just for my own understanding.